### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by mozillakab**
more translations (52%)

### CODE REVIEW

File: locales/kab_DZ/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -4605,62 +4625,62 @@ msgstr "dagi"
...
-msgstr ""
+msgstr "afrayan"
 
 #. Used for the <em>color settings</em> into the <a href="https://duckduckgo.com/params.html">URL params page</a> as the last available value for the color parameter.
-#. <em>%s</em> is for a hexadecimal color code as <em>#395323</em>, but it has to be safe encoded, and as <em>#</em> seems not safe, <em>%23</em> must be used in place of the sharp symbol, but they are the same for your browser. 
+#. <em>%s</em> is for a hexadecimal color code as <em>#395323</em>, but it has to be safe encoded, and as <em>#</em> seems not safe, <em>%23</em> must be used in place of the sharp symbol, but they are the same for your browser.
 msgid ""
 "or write out the color code you want, e.g. %s (%s is an encoded %s char)."
-msgstr ""
+msgstr "neɣ aru tangalt n yini i tebɣiḍ, md. %s (%S d asekkil %s s tengalt)."
```
Comment (andrey-p) [inline locales/kab_DZ/LC_MESSAGES/duckduckgo.po]: Can you change the `%S` to lowercase? That's why CI is complaining.

```suggestion
msgstr "neɣ aru tangalt n yini i tebɣiḍ, md. %s (%s d asekkil %s s tengalt)."
```

**Comment by mozillakab** [comment]
@andrey-p suggestion is true

**Comment by mozillakab** [comment]
@andrey-p  thanks
I will add the remaining translations newt week

**Comment by andrey-p** [comment]
Looks good to me. Thanks again for your contribution!

**Comment by mozillakab** [comment]
Thanks for merge. A new file will be  pulled within 4 or 5 days. For the future, please, how can I know that there is new strings to translate?

**Comment by andrey-p** [comment]
That's a good question. For the time being, the best thing to do is watch the repository ([see guide](https://help.github.com/en/github/receiving-notifications-about-activity-on-github/watching-and-unwatching-repositories)).

I'm afraid that means you'll see notifications for all other contributions. Maybe we can do something with the script that uploads new tokens so certain people also get notified (cc @jbarrett).

**Pull request merged**